### PR TITLE
Docs for Guillotine 5.1

### PR DESCRIPTION
**Pull request by anatol-sialitski**

### CODE REVIEW

**Review by rymsha — APPROVED** [review]

File: docs/advanced.adoc [hunk 1]
```diff
@@ -139,6 +139,144 @@ The resolution function will query contents of type post having the current auth
...
+List of properties for `schemaOptions` object:
+
+|===
+|Name | Description
+
+|applications: String or Array.<String>
+|Allowed application keys in addition to this site. Property is optional.
+
+|allowPaths:String or Array.<String>
+|Allowed content paths in addition to this site. Property is required.
```
Comment (alansemenov) [inline docs/advanced.adoc:198]: Why is this required? I guess the requirement is to use at least one of the three options?

File: docs/usage.adoc [hunk 2]
```diff
@@ -91,11 +91,41 @@ The subscriptions are implemented using Websockets, providing instant and high p
...
-  event
+  event {
+    type
+    dataAsJson
+  }
+}
+----
+
+Starting from image:images/v-500.svg[Since verision,opts=inline] Guillotine listens node events only for a specific repository and branch of the site.
+
```
Comment (alansemenov) [inline docs/usage.adoc:102]: listens node events -> listens to node events

File: docs/usage.adoc [hunk 3]
```diff
@@ -91,11 +91,41 @@ The subscriptions are implemented using Websockets, providing instant and high p
...
+.Provide `branch`, `repositoryId` and `siteId` to `data` object for WebSocket request:
```
Comment (alansemenov) [inline docs/usage.adoc:103]: "... to `data` object..." -> "... in the `data` object..."

File: docs/usage.adoc [hunk 4]
```diff
@@ -91,11 +91,41 @@ The subscriptions are implemented using Websockets, providing instant and high p
...
+        branch: req.branch,
+        repositoryId: req.repositoryId,
+        siteId: portalLib.getSite()._id
+    },
+    subProtocols: ['graphql-ws']
+}
+----
+
+or to use the `guillotineLib.createWebSocketData(req)` method which is available starting from image:images/v-5.1.0.svg[Since verision,opts=inline].
+
```
Comment (alansemenov) [inline docs/usage.adoc:120]: "or to use..." -> "or use..."

File: docs/usage.adoc [hunk 5]
```diff
@@ -91,11 +91,41 @@ The subscriptions are implemented using Websockets, providing instant and high p
...
+[source,javascript]
+----
+webSocket: {
+    data: guillotineLib.createWebSocketData(req),
+    subProtocols: ['graphql-ws']
 }
 ----
 
-Starting from image:images/v-500.svg[Since verision,opts=inline] Guillotine listens events only for a specific `repository` and `branch` which are determined from a site context.
 By default will be listened `node.*` events. In order to configure which events should be listened by application or site you must to specify the `subscriptionEventTypes` option on schema creation, in this case default config will be overwritten.
```
Comment (alansemenov) [inline docs/usage.adoc:129]: This we need to fix as well.

_Only `node.*` events are listened to by default. In order to configure which events should be listened to by an application or a site the `subscriptionEventTypes` option must be specified on schema creation, in this case default config will be overwritten._

**Review by alansemenov — CHANGES_REQUESTED** [review]

File: docs/advanced.adoc (hunk 1 above)
Comment (anatol-sialitski) [reply docs/advanced.adoc:198]: Good catch. It's my mistake. Sure, that option is optional

**Review by alansemenov — APPROVED** [review]
